avocado/plugins/yaml_to_mux.py

```diff
@@ -45,6 +46,23 @@
 __RE_FILE_SUBS = re.compile(r'(?<!\\)\\:')  # substitute '\\:' but not '\\\\:'
 
 
+class _BaseLoader(Loader):
+    """
+    YAML loader with additional features related to mux
+    """
+    pass
+
+
+_BaseLoader.add_constructor(u'!include', lambda loader,
```

Comment:
Oh actually you're right, by using the Loader's class method I can do it from the `_BaseLoader`'s scope so this patch is in deed better.